### ISSUE TITLE
Incorrect calculation of accessTokenResponse.expiresOn date

### ISSUE THREAD

**Issue by sergey-tihon** [AzureAD/microsoft-authentication-library-for-js#790]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Browser:
- [x] Chrome version XX
- [ ] Firefox version XX
- [ ] IE version XX
- [ ] Edge version XX
- [ ] Safari version XX

## Library version
1.0.2

## Current behavior
Library incorrectly calculate access token expiration date
Token should leave about 1h (not 49 years)
![Skype](https://user-images.githubusercontent.com/1197905/60192421-7a5fbe00-983e-11e9-8b7a-49fc5053f7e1.png)

## Minimal reproduction of the problem with instructions

You do have `Utils. expiresIn` that add `this.now()` to expiration window and technically return `expiresOn`.

https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/0648a237f774e71f0a10e95eeafea92c0df7b8a4/lib/msal-core/src/Utils.ts#L141-L147

but later in the code you add `Utils.now()` one more time and add extra `2019-1970=49` years to expiration.

```js
const expiresIn = Utils.expiresIn(parameters[Constants.expiresIn]).toString();
let exp = Number(expiresIn);
accessTokenResponse.expiresOn = new Date((Utils.now() + exp) * 1000);
```

here is the buggy code

https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/2fefca80827b2c408dad28d79de6f10664aa4031/lib/msal-core/src/UserAgentApplication.ts#L1714-L1724

**Comment by sameerag** [comment]
Thanks, @sergey-tihon for the fix @ #791. We will try to close loop once testing is complete and have it released asap.

**Comment by DarylThayil** [comment]
@sergey-tihon  fixed in #799 released in 1.1.1
